pkg/ebpf/processor_proctree.go

```diff
@@ -243,3 +245,38 @@ func (t *Tracee) procTreeExitProcessor(event *trace.Event) error {
 		},
 	)
 }
+
+//
+// Processors that enrich events with process tree information
+//
+
+// procTreeAddBinInfo enriches the event with processes information from the process tree.
+func (t *Tracee) procTreeAddBinInfo(event *trace.Event) error {
+	currentProcess, procOk := t.processTree.GetProcessByHash(event.ProcessEntityId)
+	if !procOk {
+		_, threadOk := t.processTree.GetThreadByHash(event.ProcessEntityId)
+		if !threadOk {
+			logger.Debugw(
+				"error enriching event executable info",
+				"pid", event.HostProcessID,
+				"tid", event.HostThreadID,
+				"event name", event.EventName,
+				"timestamp", event.Timestamp,
+				"error", "not found in process tree",
+			)
+		}
+		return nil // threads don't have executable info in the process tree
+	}
+
+	// Event timestamp is changed to relative (or not) at the end of all processors only.
+	eventTimestamp := time.Unix(0, int64(event.Timestamp)+int64(t.bootTime))
```

Comment:
How is this related to this PR?
Also - isn't it already taken care of by normalize time processor?

Reply:
Event timestamp is ns since boot and proctree timestamp is ns since epoch. The time normalization (that will convert time to be epoch or not) runs at the end of all processors (so it does not disturb what processors expect).